spec/acceptance/traits_spec.rb

```diff
@@ -20,7 +20,7 @@
           great "GREAT!!!"
         end
 
-        trait :admin do
+        trait "admin" do
```

Comment:
I could image myself "correcting" this back to a symbol to match all the other traits. Could we add a separate example that is a bit more explicit about what we are testing?